components/restservice/config/cloudify-restservice.service

```diff
@@ -4,11 +4,13 @@ Wants=network-online.target dbus.service
 After=network-online.target
 
 [Service]
+User={{ node.properties.service_user }}
+Group={{ node.properties.service_user }}
 TimeoutStartSec=0
 Restart=on-failure
 EnvironmentFile=-/etc/sysconfig/cloudify-restservice
 ExecStart=/bin/sh -c '/opt/manager/env/bin/gunicorn \
-    --pid /var/run/gunicorn.pid \
+    --pid /run/cloudify-restservice/pid \
```

Comment:
Because `cloudify-restservice` doesn't have permission to create files in `/run`